assets/stats/lib/application_details.es6

```diff
@@ -3,12 +3,13 @@ export const applicationDetails = function (data) {
     return {
       id: data.id,
       name: data.name,
-      link: createLink(data.id, type)
+      link: createLink(data, type)
```

Comment:
This comes from this stacktrace:
https://github.com/3scale/porta/blob/4e12fc08a135de509ad57683ae082a75fc5cfb2d/assets/stats/lib/application_details.es6#L1-L2
https://github.com/3scale/porta/blob/4e12fc08a135de509ad57683ae082a75fc5cfb2d/assets/stats/provider/stats_top_apps.es6#L34-L37
https://github.com/3scale/system/blob/3213fb8ef08e86df86ccf837517d30b3f446fd4c/assets/stats/lib/series.es6#L37
https://github.com/3scale/system/blob/3213fb8ef08e86df86ccf837517d30b3f446fd4c/assets/stats/lib/series.es6#L16-L18
By writing `debugger` on the last one and faking data with these steps:
1. `rake backend:fake PROVIDER_ID=2`
2. `rake jspm:dev`
3. `rake jspm:dev:server`
I could see what it received and from where 😄 Thanks @didierofrivia  💪

Reply:
This comes from this stacktrace:
https://github.com/3scale/porta/blob/4e12fc08a135de509ad57683ae082a75fc5cfb2d/assets/stats/lib/application_details.es6#L1-L2

https://github.com/3scale/porta/blob/4e12fc08a135de509ad57683ae082a75fc5cfb2d/assets/stats/provider/stats_top_apps.es6#L34-L37

https://github.com/3scale/system/blob/3213fb8ef08e86df86ccf837517d30b3f446fd4c/assets/stats/lib/series.es6#L37

https://github.com/3scale/system/blob/3213fb8ef08e86df86ccf837517d30b3f446fd4c/assets/stats/lib/series.es6#L17

By writing `debugger` on the last one and faking data with these steps:
1. `rake backend:fake PROVIDER_ID=2`
2. `rake jspm:dev`
3. `rake jspm:dev:server`
I could see what it received and from where 😄 Thanks @didierofrivia  💪